### PR TITLE
feat: add configurable message to lineage tabs

### PR DESCRIPTION
**Pull request by allisonsuarez**
Added a message component that can be configured with markdown to add additional context to the lineage data on the Up/Downstream tabs.

Screenshot 2022-11-30 at 12 01 27 PM

### CODE REVIEW

File: frontend/amundsen_application/static/js/ducks/lineage/api/v0.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function getTableLineage(
   const tableQueryParams = getQueryParams({ key, depth, direction });
 
   return axios
-    .get(`${API_PATH}/get_table_lineage?${tableQueryParams}`)
+    .get(`${API_PATH}/get_table_lineage?${tableQueryParams}`, { timeout: 90000 })
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/ducks/lineage/api/v0.ts:17]: Let's move this number into a constant

File: frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 // Copyright Contributors to the Amundsen project.
 // SPDX-License-Identifier: Apache-2.0
-
+import AppConfig from 'config/config';
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx:3]: Nit: I would move this to line 5

File: frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx [hunk 3]
```diff
@@ -55,8 +69,19 @@ export const LineageList: React.FC<LineageListProps> = ({
     );
   }
 
-  return (
+  const message = AppConfig.tableLineage.inAppListMessageGenerator &&
+    tableDetails &&
+      AppConfig.tableLineage.inAppListMessageGenerator(
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx:74]: I think you can make `AppConfig.tableLineage.inAppListMessageGenerator?.(...` instead

File: frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/styles.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+@import 'variables';
+@import 'typography';
+
+.lineage-message {
+    margin-top: -$spacer-3;
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/styles.scss:8]: Negative margins are usually a hacking thing we do from time to time.
Is there another alternative in this case?

Comment (allisonsuarez) [reply frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/styles.scss:8]: it seems like the list is what introduces that extra margin at the bottom so I can try removing that and see if that does it

File: frontend/amundsen_application/static/js/ducks/lineage/api/v0.ts (hunk 1 above)
Comment (allisonsuarez) [reply frontend/amundsen_application/static/js/ducks/lineage/api/v0.ts:17]: my bad this was a test I will clean it up

File: frontend/amundsen_application/static/css/_list-group.scss [hunk 5]
```diff
@@ -4,7 +4,7 @@
 @import 'variables';
 
 .list-group {
-  margin: 24px 0;
+  margin: 0 0;
```
Comment (Golodhros) [inline frontend/amundsen_application/static/css/_list-group.scss:7]:
```suggestion
  margin: 0;
```

File: frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx [hunk 6]
```diff
@@ -55,29 +69,42 @@ export const LineageList: React.FC<LineageListProps> = ({
...
-        const logging = {
-          index,
-          source: `table_lineage_list_${direction}`,
-        };
-        const tableResource: TableResource = {
-          ...table,
-          type: ResourceType.table,
-          description: '',
-        };
+    <div>
```
Comment (Golodhros) [inline frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.tsx:83]: This could be a fragment? 
<></>

**Review by Golodhros — APPROVED** [review]
LGTM

**Review by kristenarmes — APPROVED** [review]

**Pull request merged**